### ISSUE TITLE
Pico G2 4K slow seek times (360 4K video)

### ISSUE THREAD

**Issue by mrtnRitter** [RenderHeads/UnityPlugin-AVProVideo#832]
I'm still in the creation process, so maybe I just missed an option or I'm doing something wrong along the way. 

I have a Pico G2 4K with a H.265, 4096x4096, 100MB/s, 360 TB video which plays just fine. The video is locally on the headset. 

When I seek in the timeline, it takes up to 5 seconds until the video is playing again. The "buffered bar" is filling (first buffered segment is quiet large, the other chunks are smaller), which is a bit odd on local videos? If I seek into the buffered area of the timeline, all buffered data disappear and buffering starts again, which seems wrong. 

If I use the build in video player from Pico, I can seek through the video with no delay. 
Also, if I seek through in the Unity Editor (Windows, video file is on network drive, 1GBit link speed), there is no delay either. I can't even see the buffered bar. 

Is there anything I can do to lower the buffering time, or is this somehow a limit on how everything works?

Thanks again!

**Comment by RenderHeadsSte** [comment]
What version of AVPro Video are you using @mrtnRitter

**Comment by mrtnRitter** [comment]
Hi RenderHead!

I'm on 2.1.3 - trial and the video is H.265, MP4. 

If this is a limit of the trial version - no problem, I was going to get the android core edition anyways.

**Comment by RenderHeadsSte** [comment]
No, this is not because of the trial version.  The trail is essentially the Ultra Edition with watermarking (and a few locked off features - such as OES)

You could try changing the buffering values (screen shot attached), though I think they only have an effect when using a streaming format.  Worth a go though.

![image](https://user-images.githubusercontent.com/46531149/123270766-1be67c80-d4f8-11eb-928f-16dbf0405eee.png)

**Comment by mrtnRitter** [comment]
Hi again. 

I tried those settings, but they hadn't any affect, just the buffer bar was not so huge anymore. 

I'll send you one minute of the video via wetransfer. 

Thank you very much!

**Comment by RenderHeadsSte** [comment]
Your video has keyframes every 6.66 seconds...

![image](https://user-images.githubusercontent.com/46531149/123303160-67f3ea00-d515-11eb-81d8-700e8a291bdb.png)

I re-encoded it with keyframes every second (and the moov atom at the start, as you already have) and at 225MBit (down to a random CRF value I chose).  I also set all the above buffering values to 0 (as it does make a difference).

The result is a lot faster seeking.  Maybe give a re-encode a try with more keyframes?

(here is the ffmpeg line I used for the re-encode: ffmpeg.exe -i LTS_VRTour_Standalone_DE.mp4 -vcodec libx264 -x264-params keyint=30:scenecut=0 -c:a copy -crf 8 -movflags faststart LTS_test.mp4)
